packages/components/src/Lettera.purs

```diff
@@ -14,7 +14,7 @@ import Data.Date (Date, day, month, year)
 import Data.Either (Either(..), either, hush, isRight)
 import Data.Enum (fromEnum)
 import Data.Foldable (class Foldable, foldMap)
-import Data.Foldable as Foldable
+import Data.Foldable (foldl, foldr) as Foldable
```

Comment:
Looks like unnecessary change.

Reply:
I thought it would be helpful for the developer to see which functions we are using from `Data.Foldable`. On the other hand, maybe it just creates unnecessary diff noise.